mlir/utils/jenkins/Dockerfile

```diff
@@ -118,6 +119,11 @@ RUN apt-get update && rm /usr/share/keyrings/kitware-archive-keyring.gpg && \
   apt-get clean && \
   rm -rf /var/lib/apt/lists/*
 
+# Install the new rocm-cmake version
```

Comment:
Why is this necessary? I think just get the rocm-cmake from default ROCm distro is fine?

Reply:
This is what CK does, probably because the default ROCM doesn't include some newest updates.

Reply:
I just took a look at the master branch, it is 5 months old now. `develop` branch is actually newer than it. I'd say get rid of those line and give it a shot to see this is really necessary on our end.

Reply:
I think what's going on here is that `rocm-cmake` has the CMake files you need to do a ROCm build, which aren't installed by default because most people don't need them?

Reply:
I may be wrong but I thought its exported CMake files are the key to make it useful to compile other components.

The worry I have is that we'd always install a stale rocm-cmake with the manual steps, and hardly notice it until it implicitly break us in the ROCm upgrade.

Reply:
`rocm-cmake` doesn't get frequently updated, and these days we're making a strong effort to ensure that it's backwards compatible. The version of `rocm-cmake` installed with ROCm 5.4 (which is `rocm-cmake` 0.8.0 IIRC) should be sufficient for you folks.

Reply:
OK, I'll remove the manual build and installation. Thanks!